### PR TITLE
Add KTX support

### PR DESCRIPTION
**Pull request by THISISAGOODNAME**

### CODE REVIEW

File: src/Modules/Texture/TextureLoader.cpp [hunk 1]
```diff
@@ -47,10 +47,33 @@ std::shared_ptr<Resource> CreateSRVFromFileDDS(RenderDevice& device, RenderComma
     return res;
 }
 
+std::shared_ptr<Resource> CreateSRVFromFileKTX(RenderDevice& device, RenderCommandList& command_list, const std::string& path)
```
Comment (andrejnau) [inline src/Modules/Texture/TextureLoader.cpp:50]: What is the difference between CreateSRVFromFileDDS and CreateSRVFromFileKTX? They look exactly the same.

Comment (THISISAGOODNAME) [reply src/Modules/Texture/TextureLoader.cpp:50]: Yes, they are the same. 
GLI provides classes and functions to load image files (KTX and DDS)

Comment (andrejnau) [reply src/Modules/Texture/TextureLoader.cpp:50]: So, if they are the same, what is the reason to have them? Could you please just rename CreateSRVFromFileDDS and use it for dds as well as for ktx.

Comment (THISISAGOODNAME) [reply src/Modules/Texture/TextureLoader.cpp]: Done.

**Pull request merged**